dev/Scroller/APITests/ScrollerLayoutTests.cs

```diff
@@ -306,8 +306,8 @@ public void ViewportHeight()
             VerticalAlignment scrollerVerticalAlignment,
             double scrollerContentHeight)
         {
-            Log.Comment("ViewportHeight test case - isScrollerParentSizeSet: " + isScrollerParentSizeSet + ", isScrollerParentMaxSizeSet: " + isScrollerParentMaxSizeSet +
-                ", scrollerVerticalAlignment: " + scrollerVerticalAlignment + ", scrollerContentHeight: " + scrollerContentHeight);
+            Log.Comment("ViewportHeight test case - isScrollerParentSizeSet: {0}, isScrollerParentMaxSizeSet: {1}, scrollerVerticalAlignment: {2}, scrollerContentHeight: {3}",
```

Comment:
Oh, it gets even better than that:
```
Log.Comment($"ViewportHeight test case - isScrollerParentSizeSet: {isScrollerParentSizeSet}, isScrollerParentMaxSizeSet: {isScrollerParentMaxSizeSet}, scrollerVerticalAlignment: {scrollerVerticalAlignment}, scrollerContentHeight: {scrollerContentHeight}"); 
```
Note the $ at the start of the string.

Reply:
Oh sweet. Done!